lib/components/dashboard/course-list/horizontal/Semester.tsx

```diff
@@ -195,8 +195,10 @@ const Semester: FC<{
    */
   const updateDistributions = (): void => {
     if (version !== 'None') {
+      console.log(version);
```

Comment:
Please remove the console log ^.^

Reply:
OOPS My mistake

Reply:
I will clean up the code when I submit a second PR after I fix CS major. I only submitted a PR urgently so that people can start using the new pathing implementation  :)

Reply:
sgsg